code/modules/client/preferences.dm

```diff
@@ -81,8 +81,11 @@ datum/preferences
 				return
 
 		stage = "load"
-		load_preferences()
-		load_character()
+		for(var/i=1, i < 5, i++) //hack hack BAD BAD!
```

Comment:
There's no sleeping between iterations and as far as I'm aware isn't byond single threaded meaning this wouldn't load right?